lib/pharos/scripts/pharos.sh

```diff
@@ -62,12 +62,14 @@ configure_container_runtime_proxy() {
   systemd_proxy_config_file="${systemd_runtime_dir}/http-proxy.conf"
   mkdir -p "${systemd_runtime_dir}"
   # shellcheck disable=SC2154
-  if [ -n "$HTTP_PROXY" ] || [ -n "$HTTPS_PROXY" ] || [ -n "$NO_PROXY" ] || [ -n "$http_proxy" ] ||  [ -n "$FTP_PROXY" ]; then
+  if [ -n "$HTTP_PROXY" ] || [ -n "$HTTPS_PROXY" ] || [ -n "$NO_PROXY" ] || [ -n "$http_proxy" ] || [ -n "$https_proxy" ] || [ -n "$no_proxy" ] || [ -n "$FTP_PROXY" ]; then
```

Comment:
You can probably replace the whole thing with something like:
```
(env | cut -d"=" -f1|grep -i -- "_proxy$") | while read -r var; do 
  [ -n "${!var}" ] && echo "Environment=\"${var}=${!var}}\"" >> "${systemd_proxy_config_file}"
done
```

Reply:
But I guess it's fine the way it is. It can be beautified at a later time if desired.

Reply:
I think the reason for this is line 66:
```
      echo "[Service]" > "${systemd_proxy_config_file}"
```

We'd have to check if one of the variables is set, but the inner part can be enhanced.